cdap-ui/app/features/apps/templates/detail.html

```diff
@@ -25,7 +25,7 @@
                 tooltip-append-to-body="true"></span>
         </span>
         <span ng-if="DetailController.myHydratorFactory.isETLApp(DetailController.artifact.name)">
-          <span ng-if="DetailController.artifact.name === DetailController.GLOBALS.etlBatch">
+          <span ng-if="[DetailController.GLOBALS.etlBatch, DetailController.GLOBALS.etlDataPipeline].indexOf(DetailController.artifact.name) !== -1">
```

Comment:
I think instead of doing the check in here, we should declare this array as a constant in GLOBALS as well